### ISSUE TITLE
split prompt bug?

### ISSUE THREAD

**Issue by BornSaint** [rsxdalv/tts-generation-webui#225]
looks like split prompt by lines option is increasing the seed after line generation, because the voice is changing after each line.

**Comment by rsxdalv** [comment]
Yes it does that, I don't remember if there's a way to keep a static seed
in the UI. It's not that big of a change to "break" this feature and make
it always return the same seed if you can find the function. Even so, for
general usage I recommend using voice files rather than just seeds to get a
particular "character" as the consistency will be much better.

**Comment by 520Pig520** [comment]
Wait, you're the programmer, you can't tell where i can find this function? Also, what you mean by use voice files? I'm already using a voice file to generate, and i didn't notice any option to keep static seed, can you implement this please? or at least show where i can modify the code to implement?

**Comment by 520Pig520** [comment]
https://i.imgur.com/nSHN8Ax.png

the solution is ignore the index?

**Comment by rsxdalv** [comment]
I'm on an airplane. No I cannot tell you where to find this function.

**Comment by rsxdalv** [comment]
> https://i.imgur.com/nSHN8Ax.png
> 
> the solution is ignore the index?

Yes, that is the solution.
